spec/engine/miq_ae_method_service/miq_ae_service_model_base_spec.rb

```diff
@@ -68,6 +68,12 @@
       expect(described_class.create_service_model_from_name(:VmOrTemplate)).to be_nil
     end
 
+    it 'shouldn\'t expose private attr' do
+      allow(MiqAeMethodService).to receive(:const_get)
+      expect(MiqAeMethodService::MiqAeServiceModelBase).not_to receive(:expose).with('properties')
```

Comment:
@pkomanek 
Can we add a spec to ensure that the methods function call doesn't list the method. like it is mentioned in the BZ.